kaggle_environments/envs/halite/halite.py

```diff
@@ -86,6 +86,11 @@ def populate_board(state, env):
     size = env.configuration.size
     uid_counter = 0
 
+    # Set seed for random number generators
+    if hasattr(config, "randomSeed"):
```

Comment:
Using `null` is JSON works however the default behavior is that there is no attribute in the struct, so `hasattr()` is needed. Works for you (alternative would be to create an attribute with value `None`.